app/src/main/res/layout/fragment_pledge.xml

```diff
@@ -172,19 +172,71 @@
         </LinearLayout>
 
         <TextView
+          android:id="@+id/shipping_rules_section_text_view"
           style="@style/CalloutPrimaryMedium"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
+          android:layout_marginBottom="@dimen/grid_1"
           android:text="@string/Your_shipping_location" />
 
-        <TextView
-          android:id="@+id/shipping_amount"
-          style="@style/PledgeCurrency"
-          android:layout_width="wrap_content"
+        <LinearLayout
+          android:id="@+id/shipping_rules_row"
+          android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_gravity="end"
           android:layout_marginBottom="@dimen/grid_4"
-          tools:text="$20" />
+          android:gravity="center_vertical"
+          android:orientation="horizontal"
+          android:weightSum="1.8">
+
+          <AutoCompleteTextView
+            android:id="@+id/shipping_rules"
+            style="@style/AutocompleteStyle"
+            android:layout_width="@dimen/autocomplete_text_view_width"
+            android:layout_height="wrap_content"
+            android:layout_weight=".7"
+            android:dropDownAnchor="@id/shipping_rules_row"
+            android:ellipsize="end"
+            android:enabled="false"
+            android:hint="@string/Shipping"
+            android:maxLines="2"
+            android:scrollHorizontally="true"
+            android:text="@string/Loading"
+            android:textColor="@color/ksr_green_500" />
+
+          <Space
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_weight="1" />
+
+          <androidx.appcompat.widget.AppCompatImageView
+            android:layout_width="@dimen/plus_sign_width"
+            android:layout_height="@dimen/plus_sign_height"
+            android:backgroundTint="@color/ksr_grey_500"
+            android:src="@drawable/ic_add"
```

Comment:
It's too small and it's the wrong color.

<img src=https://user-images.githubusercontent.com/1289295/58514077-f21ec800-816e-11e9-8116-6cf5f9b13cc0.png width=330/>

Reply:
I fixed the color! Also the abstract has properties of `20dp for height` and `9dp for width`

https://app.goabstract.com/projects/ff85f7f0-23b6-11e8-9c12-3d345000cd85/branches/master/commits/4cd104da8a116335c9a2861a888a0bdba50a261d/files/DD4807BD-4D34-43F3-853E-4765774306D1/layers/426005EF-46B4-47F2-B95E-45C092D754AB?mode=build&nonce=2887376e-9c5f-4696-b33b-f0729f16e0bf&selected=3355706653-61DBAB2D-6F81-458E-B42B-3B6C7EA9854C

Reply:
A drawable has intrinsic padding so when you said a fixed size and not a scaletype, it doesn't take that into account. It's too small.

Reply:
The positioning is also off.